Rulebook.tex

```diff
@@ -62,27 +62,29 @@
 \chapter{Tests in Stage I}
 \label{chap:stage_I}
 
-\section*{Housekeeper tasks}
-\testtocentry{test:clean-up}
-\testtocentry{test:gpsr}
+\section*{Housekeeper}
+\testtocentry{test:tidy-up}
 \testtocentry{test:storing-groceries}
+\testtocentry{test:gpsr}
 
 
-\section*{Party Host tasks}
+\section*{Party Host}
 \testtocentry{test:carry-my-luggage}
 \testtocentry{test:receptionist}
 
 % Uncomment to get aesthetic improvement
-\cleardoublepage
-\input{tasks/CarryMyLuggage}
 \newpage
-\input{tasks/CleanUp}
+\input{tasks/TidyUp}
```

Comment:
Why change the order? It's not clear from the titles alone why they're ordered this way (it's because of the themes, but the themes aren't listed anywhere so I don't think the reader would get it).

Reply:
Maybe they're now ordered in increasing difficulty? Or just moving GPSR to the end out of tradition.

Reply:
I believe they were just ordered inconsistently. The order in the toc was different to the order in the text and I just happened to make the TOC fit the text. No real reason for the order.

Reply:
I think the reasoning for the old TOC being out of order was that, if you're flipped to it, you just read the page number for the task you want (or click on it) and it's fine if that number is out of sequence. The TOC doesn't even need to be alphabetized really because there aren't many entries. What's more important is that the tasks are ordered alphabetically in page sequence, because if you open to Carry my luggage, you can then know for sure which direction you need to page/scroll to get to GPSR.

Reply:
Ordered alphabetically